src/api/queries/blockchainQueries.js

```diff
@@ -158,3 +158,20 @@ ApiClient_.prototype.fetchRealizedValue = function (slug, from, to) {
 
   return this.conn.graphQLQuery(query, 'realizedValue')
 }
+
+ApiClient_.prototype.fetchHistoricalBalance = function (slug, from, to, address) {
+  var query = {
+    'query': '{\
+       historicalBalance(slug: "' + slug + '",\
+                         from: "' + toUTC_(from) + '",\
+                         to: "' + toUTC_(to) + '",\
+                         interval: "1d"\
```

Comment:
missing comma here

Reply:
I thought that I've made a stubbed test for this one, I will make it now, thanks for the heads-up, also for the comma

Reply:
Seems that commas are not needed, maybe it's better to remove them in a separate PR. But add one here for now.